play_tester.py

```diff
@@ -9,15 +9,17 @@
 pp = pprint.PrettyPrinter(indent=4)
 
 # play testing section
-print("\n-----Wave 01 test data-----")
-pp.pprint(HORROR_1)
-pp.pprint(FANTASY_1)
-pp.pprint(FANTASY_2)
+# print("\n-----Wave 01 test data-----")
+# pp.pprint(HORROR_1)
+# pp.pprint(FANTASY_1)
+# pp.pprint(FANTASY_2)
 
 # print("\n-----Wave 02 user_data-----")
 # pp.pprint(clean_wave_2_data())
 
 #print("\n-----Wave 03 user_data-----")
+# print(get_unique_watched(USER_DATA_3))
+get_unique_watched(USER_DATA_3)
```

Comment:
In the future, if you make changes to a sandbox file like this one, you don't need to add it to the staging area/commit it and push up the changes.

Totally ok that you did so here since this isn't a PR that will get merged into the project, but if your changes were to be merged into a project then someone reviewing your code would probably ask you to remove these changes from the PR so they don't get merged into the master code base.